packages/app/src/components-styled/line-chart/index.tsx

```diff
@@ -83,44 +124,130 @@ export function LineChart<T extends Value>({
     [signaalwaarde]
   );
 
-  const trendData = useMemo(
-    () => getTrendData(values, metricProperties[0], timeframe),
+  const trendsList = useMemo(
+    () => getTrendData(values, metricProperties, timeframe),
     [values, metricProperties, timeframe]
   );
 
   const xDomain = useMemo(() => {
-    const domain = extent(trendData.map((d) => d.__date));
+    const allData = trendsList.flat();
+    const domain = extent(allData.map((d) => d.__date));
 
     return isDefined(domain[0]) ? (domain as [Date, Date]) : undefined;
-  }, [trendData]);
+  }, [trendsList]);
+
+  const yDomain = useMemo(() => [0, calculateYMax(trendsList, signaalwaarde)], [
+    trendsList,
+    signaalwaarde,
+  ]);
+
+  const bisect = useCallback(
+    (
+      trend: (T & TrendValue & Value)[],
+      xPosition: number,
+      xScale: ScaleTime<number, number>
+    ) => {
+      if (!trend.length) return;
+      if (trend.length === 1) return trend[0];
+
+      const date = xScale.invert(xPosition - padding.left);
+
+      const index = bisectLeft(
+        trend.map((x) => x.__date),
+        date,
+        1
+      );
 
-  const yDomain = useMemo(
-    () => [0, calculateYMax(trendData, metricProperties, signaalwaarde)],
-    [trendData, metricProperties, signaalwaarde]
+      const d0 = trend[index - 1];
+      const d1 = trend[index];
+
+      if (!d1) return d0;
+
+      return +date - +d0.__date > +d1.__date - +date ? d1 : d0;
+    },
+    [padding]
   );
 
-  const handleHover = useCallback(
+  const distance = (point1: HoverPoint<unknown>, point2: Point) => {
+    const x = point2.x - point1.x;
+    const y = point2.y - point1.y;
+    return Math.sqrt(x * x + y * y);
+  };
+
+  const toggleHoverElements = useCallback(
     (
-      event:
-        | React.TouchEvent<SVGRectElement>
-        | React.MouseEvent<SVGRectElement>,
-      data,
-      xPosition,
-      yPosition
+      hide: boolean,
+      hoverPoints?: HoverPoint<T>[],
+      nearestPoint?: HoverPoint<T>
     ) => {
-      if (event.type === 'mouseleave') {
+      if (hide) {
         hideTooltip();
-      } else {
+        setMarkerProps(undefined);
```

Comment:
```suggestion
        setMarkerProps();
```

This would be considered the same

Reply:
The compiler disagrees I'm afraid :)